src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -54,4 +59,12 @@ public String getPaymentRepaymentOption() {
     public String getAmountToPayByDefendant() {
         return amountToPayByDefendant;
     }
+
+    public String getClaimantName() {
+        return claimantName;
+    }
+
+    public String getCcjRequestedDate() {
```

Comment:
That should have been reflected in `uk.gov.hmcts.cmc.claimstore.services.staff.content.CCJContentProviderTest`

Reply:
why? we don't usually check all the get and set values.